src/credentials/CHIPOpCred.h

```diff
@@ -0,0 +1,184 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file defines data types and objects for modeling and
+ *      working with CHIP Operational Credentials.
+ *
+ */
+
+#pragma once
+
+#include <credentials/CHIPCert.h>
+#include <crypto/CHIPCryptoPAL.h>
+#include <support/DLLUtil.h>
+
+namespace chip {
+namespace Credentials {
+
+using namespace Crypto;
+
+/**
+ *  @class ChipOperationalCredentialSet
+ *
+ *  @brief
+ *    Collection of CHIPCertificateSet providing methods for
+ *    finding and validating Device Credentials against Roof of Trust chains.
+ */
+class DLL_EXPORT ChipOperationalCredentialSet
```

Comment:
Let's remove `Chip` from the class name. The namespace is already accounting for it.

Reply:
Maybe even from the header name. The project is chip - although we inherited some naming starting with CHIP they seem redundant.